jcstress-core/src/main/java/org/openjdk/jcstress/infra/runners/TestConfig.java

```diff
@@ -244,4 +245,23 @@ public void generateDirectives(PrintWriter pw, Verbosity verbosity) {
         pw.println("]");
         pw.flush();
     }
+
+
+    public String toDetailedTest() {
+        //binaryName have correct $ instead of . in name; omitted
+        //generatedRunnerName name with suffix (usually _Test_jcstress) omitted
+        //super.toString() as TestConfig@hash - omitted
+        return name +
+                " {" + actorNames +
+                ", spinLoopStyle=" + spinLoopStyle +
+                ", threads=" + threads +
+                ", forkId=" + forkId +
+                ", maxFootprintMB=" + maxFootprintMB +
+                ", compileMode=" + compileMode +
+                ", shClass=" + shClass +
+                ", strideSize=" + strideSize +
+                ", strideCount=" + strideCount +
+                ", cpuMap=" + cpuMap +
+                ", " + jvmArgs + "}";
```

Comment:
kindly modify it to return the json based structure, as followed in rest of the class, equals symbol does not appear in json

Reply:
fixed